libraries/ESP8266mDNS/src/LEAmDNS.cpp

```diff
@@ -640,7 +640,7 @@ uint32_t MDNSResponder::queryService(const char* p_pcService,
         if (_sendMDNSServiceQuery(*pServiceQuery)) {
             // Wait for answers to arrive
             DEBUG_EX_INFO(DEBUG_OUTPUT.printf_P(PSTR("[MDNSResponder] queryService: Waiting %u ms for answers...\n"), p_u16Timeout););
-            delay(p_u16Timeout);

```

Comment:
@LaborEtArs @hreintke 
Is this delay supposed to be interrupted by `esp_schedule()` ?
(I don't think so)

Ideally, should there be instead a timer started here and that would be tested later ?

Reply:
After some time of thinking it's seems to me that we should leave generic delay() in mDNS code. As delay is not required to be interrupted -- it should lasting to get responses from [unknown count of] multiple mDNS actors.